test/legacy-upgrade/mzcompose.py

```diff
@@ -124,14 +124,13 @@ def test_upgrade_from_version(
                 )
             )
 
-    priors = priors + prior_patch_versions
-    priors.sort()
-    priors = [f"{prior}" for prior in priors]
+    priors = sorted(priors + prior_patch_versions)
+    prior_strings = [repr(prior) for prior in priors]
```

Comment:
Why did this change from an `f` string to `repr`?

Reply:
No strong reason, just a preference a simple function call when the `f`-string was only there to stringify the value.